tests/apollo/test_skvbc_byzantine_primary_preexecution.py

```diff
@@ -25,7 +25,7 @@
 BATCH_SIZE = 4
 NUM_OF_SEQ_WRITES = 25
 
-def start_replica_cmd(builddir, replica_id, view_change_timeout_milli="10000"):
+def start_replica_cmd(builddir, replica_id, view_change_timeout_milli="6000000"):
```

Comment:
Seriously,  1,5h ? 😄

Reply:
Thanks @teoparvanov for reviewing the changes.
Yes, this I kept to test the max limit. Now the test case at max took 40 - 50 secs, at peak load today and during that time this high expiry prevented any of the view change. Tomorrow I will make it to 10 mins and check. If it works fine in the peak load, then I will keep that 10 mins. Although keeping it for 1 hr 40 mins is not a problem.

Reply:
I don't understand what you're trying to achieve with this... The test expects a view change to happen, but you set a large timeout,  effectively disabling the view change protocol? Did you check the replica logs to verify that a view change is triggered correctly?

Reply:
@teoparvanov as discussed, increasing view change timeout in milli sec is not needed. We are reducing the total number of requests sent. Thanks for your review.